### ISSUE TITLE
Using n-gon as a base for a science project

### ISSUE THREAD

**Issue by AnnonymousNerd87** [landgreen/n-gon#259]
I am doing a science project on nuclear energy and I want to use n-gon as a base for the movement and code, but don't want too much of n-gon's features so it will be easier to present and show what I did. I already set the tech frequency for all the tech to 0, with the exception of a placeholder tech I made, and next I'm going to remove the ability to get guns other than foam, (bc foam is easy to work with and I can change the color and stuff of the particles to fit the project) but am stuck because guns don't use the same frequency system that tech uses. how can I ensure that foam is the only option? I already set the number of visible choices to 1, but I can't figure out how to set the random function to always be 8 for foam.

**Comment by AnnonymousNerd87** [comment]
tldr; How can I set foam to be the only possible gun without deleting all the other guns

**Comment by Masbirb** [comment]
Maybe on start you are given a tech that gives you foam and then set gun pickup frequency to 0

**Comment by kgurchiek** [comment]
If you don't want to delete the other guns couldn't you just find a seed that gives foam in the first gun powerup?

**Comment by AnnonymousNerd87** [comment]
I would like it to be in the code, like maybe setting the random value for
gun pickups to always be foam.

**Comment by kgurchiek** [comment]
The gun is spawned in the spawnStartingPowerUps function in powerup.js on line 1459.

**Comment by AnnonymousNerd87** [comment]
got it
